### PR TITLE
feat(web): redis for session storage

### PR DESCRIPTION
**Pull request by beejjacobs**
## Describe your changes

Use Redis for session storage, instead of the in memory store (which isn't intended for production).

Existing local config will work, using the in memory store.

Corresponding infra PRs:
https://github.com/Planning-Inspectorate/infrastructure-environments/pull/431
https://github.com/Planning-Inspectorate/infrastructure-environments/pull/432
https://github.com/Planning-Inspectorate/infrastructure-environments/pull/434

## Issue ticket number and link

[BOAS-1034](https://pins-ds.atlassian.net/browse/BOAS-1034)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

[BOAS-1034]: https://pins-ds.atlassian.net/browse/BOAS-1034?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: appeals/web/src/server/app/config/session.js [hunk 1]
```diff
@@ -1,10 +1,68 @@
-import config from '@pins/appeals.web/environment/config.js';
+import config from '@pins/applications.web/environment/config.js';
```
Comment (fabfab-kc) [inline appeals/web/src/server/app/config/session.js:1]: should be from `appeals`?

**Review by fabfab-kc — APPROVED** [review]
LGTM! in-memory client for local dev and dist session on deployments...
I think there's a mismatch between appeals / apps, reported in a comment.

File: appeals/web/src/server/app/config/session.js (hunk 1 above)
Comment (fabfab-kc) [reply appeals/web/src/server/app/config/session.js:1]: sorry just noticed this is running from `apps/api`... Should we duplicate or maybe move to packages if possible?

Comment (beejjacobs) [reply appeals/web/src/server/app/config/session.js:1]: Good catch!

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**